libraries/rtos/rtx/TARGET_CORTEX_A/rt_CMSIS.c

```diff
@@ -905,7 +905,7 @@ uint8_t osThreadGetState (osThreadId thread_id) {
   if (__exceptional_mode()) return osErrorISR;     // Not allowed in ISR
 
   ptcb = rt_tid2ptcb(thread_id);                // Get TCB pointer
-  if (ptcb == NULL) return osErrorParameter;
+  if (ptcb == NULL) return INACTIVE;
```

Comment:
is this correct? Why this function returns either osValue or state which is different declaration.